datasets/cellpainting-gallery.yml

```diff
@@ -7,12 +7,15 @@ Contact: cellpainting@broadinstitute.org
 ManagedBy: [Carpenter-Singh Lab](https://carpenter-singh-lab.broadinstitute.org/)"
```

Comment:
and Cimin Lab

Reply:
Also, let's drop "All data generated from the [JUMP Cell Painting Consortium](https://jump-cellpainting.broadinstitute.org/) will be deposited here." because it is more than JUMP now